### PR TITLE
Update grpc-client.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@JunTaoLuo  see my comments in the file
Tutorials shouldn't have multiple options. Select what you think is the best approach. Recommend you removed this approach.

**Comment by Rick-Anderson** [comment]
@JunTaoLuo please review so I can merge into #11937

File: aspnetcore/tutorials/grpc/grpc-client.md [hunk 1]
```diff
@@ -30,50 +30,53 @@ In this tutorial, you:
...
+* Run the following command:
 
     ```powershell
     Install-Package Grpc.Core
     ```
 
-  * Repeat for Google.Protobuf and Grpc.Tools
+* Repeat the `Install-Package` for Google.Protobuf and Grpc.Tools
+
+<!-- Tutorials shouldn't have multiple options. Select what you think is the best approach. Recommend you removed this approach. -->
```
Comment (JunTaoLuo) [inline aspnetcore/tutorials/grpc/grpc-client.md]: I copied this from https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/getting-started-with-swashbuckle.md#package-installation. But if I were to pick, we can just go with the Manage Nuget Packages option.

**Comment by guardrex** [comment]
#11937

:point_up: yep ... that's a bunch of them I'm sure.

File: aspnetcore/tutorials/grpc/grpc-client.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/tutorials/grpc/grpc-client.md]: Lets leave both for now and maybe have @tdykstra  update this in the future for our editorial rules..

**Pull request merged**